### DISCUSSION TITLE
Preventing Scheduled Events to Overlap

### DISCUSSION THREAD

**Discussion by richardskinner** [nhn/tui.calendar#1094] [Q&A]
Hi,

I"m trying to add new schedules to my calendar but I don't want anything to overlap e.g. event 1 is at 0900-1000 and event 2 wants to be 0930-1030. I want event 2 not to be created as this time slot is occupied or part occupied. I understand on some of the other calendar like FullCalendar has a config item for this but I can't find this for TUI Calendar.

As you can see in the screenshot the overlap that happens.
<img width="507" alt="Screenshot 2022-04-30 at 10 47 11" src="https://user-images.githubusercontent.com/10455827/166106251-eda2c998-c519-44b6-856d-422c62a3577b.png">

I have tried using the event `beforeCreateSchedule` but couldn't find a way to get the existing scheduled items to compare against.

Any help will be highly appreciated.

Thank you.

**Comment by adhrinae** [comment]
@richardskinner
Do you mean [the background event as in Fullcalender](https://fullcalendar.io/docs/background-events)?

I'm afraid that We don't offer this feature yet.
